subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/AdbDevice.kt

```diff
@@ -38,9 +38,19 @@ data class AdbDevice(
 ) : Device {
 
     override val api: Int by lazy {
-        loadProperty(
-            key = "ro.build.version.sdk",
-            cast = { it.toInt() }
+        retry(
+            retriesCount = 5,
+            delaySeconds = 3,
+            block = { attempt ->
+                log("Attempt $attempt: getting ro.build.version.sdk")
+                loadProperty(
+                    key = "ro.build.version.sdk",
+                    cast = { it.toInt() }
+                )
+            },
+            attemptFailedHandler = { attempt, throwable ->
+                log("Attempt $attempt: reading ro.build.version.sdk failed. Reason: ${throwable.message}")
+            }
```

Comment:
This is not a solution to the problem. Anyway, it's better to retry all unstable operations.